src/wallets/solana/index.ts

```
@@ -191,8 +191,11 @@ export class SolanaWalletToolbox extends WalletToolbox {
   }
 
   public async getGasPrice () {
-    // TODO: implement
-    throw new Error('SolanaWalletToolbox.getGasPrice not implemented.');
+    // see more on why use getRecentPrioritizationFees: https://docs.solana.com/transaction_fees#get-recent-prioritization-fees
+    const recentBlocks = await this.connection.getRecentPrioritizationFees();
+    const prioritizationFee = findMedian<RecentPrioritizationFees>(recentBlocks, (block) => block.prioritizationFee);
+
+    return BigInt(prioritizationFee || 0n);
```

I think that if we didn't find a gasPrice, we should throw an error instead of returning `0n`.
This way it's up to the caller if they want to default it or throw an error (there might be cases where a 0 for gas price is un-acceptable for the client of the library, but because we fail silently they won't realise).
The same applies for Sui (and EVM in case it's returning a 0 now, didn't check)

---

Sure, I will update the methods in all walletToolBox implementations in `evm`, `solana` and `sui` to throw an error instead of returning `0`.